### PR TITLE
added new animal: Vulture

### PR DESCRIPTION
**Pull request by ryoung3**
Added new animal: Vulture

### CODE REVIEW

File: animalfacts.py [hunk 1]
```diff
@@ -2866,6 +2867,23 @@ def animalfactsbot(reddit):
...
+    'Vultures eat as much as they can at one meal. They never know when the next meal will come.',
+    'Vultures live in every part of the world except Australia and the Antarctica.',
+    'To eat their prey vultures have sharp hooked beaks and talons. They can also use tools.',
+    'Vultures sometimes drop eggs to break them or hit them against rocks to open them for food.',
+    'Vultures often appear when an animal is dying or dead. Egyptians and Native Americans used vultures in burial ceremonies.',
+    'Unlike many raptors, vultures are relatively social and often feed, fly or roost in large flocks. ',
+    'Vultures have excellent senses of sight and smell to help them locate food, and they can find a dead animal from a mile or more away.',
+    'While vultures eat mostly dead animals, they are capable of attacking and will often prey on extremely sick, wounded or infirm prey.',
+    'Vultures urinate on their legs and feet to cool off on hot days, a process called urohydrosis.',
+    'When threatened, vultures vomit to lighten their body weight so they can escape more easily into flight. '
```
Comment (mpentler) [inline animalfacts.py]: Could you pop a quick comma at the end of this line to match the other blocks? Otherwise these are all great facts!

Comment (ryoung3) [reply animalfacts.py]: No problem.
Yeah, they are !!!

**Comment by ryoung3** [comment]
Realigned the closing parenthesis as well. Sorry about that.

**Comment by mpentler** [comment]
I am learning so much! Maybe having 50 emails about PRs WASN'T a bad thing :)

**Pull request merged**